content/docs/conceptual-guides/branches.md

```diff
@@ -1,10 +1,14 @@
 ---
 title: Branches
+redirectFrom:
+  - /docs/cloud/concepts/
```

Comment:
not sure about this.
why do we want to redirect from cloud concepts here?

Reply:
Notes from the conversation today:
Add Branching to the Glossary (without removing the conceptual guides) and then move this redirection (of `/docs/cloud/concepts/`) to Glossary page.

Reply:
Hi @stepashka thank you, we will make sure to do this.